src/components/event-details/EventDetails.tsx

```diff
@@ -1,11 +1,26 @@
 import { Button, Col, Row, Typography } from 'antd';
 import dateFormat from 'dateformat';
-import React from 'react';
+import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
 import styled from 'styled-components';
-import { EventProps } from '../../containers/upcoming-events/ducks/types';
+import EventRegistrationModal from '../modals/event-registration-modal/EventRegistrationModal';
+import { DEFAULT_IMAGE } from '../../utils/copy';
+import { PrivilegeLevel } from '../../auth/ducks/types';
 const { Title } = Typography;
 
+interface EventDetailsProps {
+  id: number;
+  title: string;
+  thumbnail?: string;
+  details: {
+    description: string;
+    location: string;
+    start: Date;
+    end: Date;
+  };
+  privilegeLevel: PrivilegeLevel;
+}
+
```

Comment:
This should already be located in `upcoming-events/ducks/types`. Also, like @vrautela mentioned in my PR (that I'm about to merge), we might have some merge conflict here

Reply:
And by "here" I meant where the component is declared